src/components/Form.tsx

```diff
@@ -0,0 +1,119 @@
+import { HTMLAttributes, useState } from "react";
+import { Card } from "./atomic";
+import TitledInput from "./TitledInput";
+const _ = require("lodash"); // Is this right?
+
+// TYPES
+export type RecursivePartial<T> = {
+  [P in keyof T]?: RecursivePartial<T[P]>;
+};
+
+export type QuestionBase = {
+  readonly id: string;
+  title: string;
+  description: string;
+};
+
+export type TextQuestion = QuestionBase & {
+  type: "short-answer" | "long-answer";
+};
+
+export type MultipleChoiceQuestion = QuestionBase & {
+  type: "multiple-choice";
+  choices: Set<string>; // TODO: Multiple choice data structure
+};
+
+export type Question = MultipleChoiceQuestion | TextQuestion;
+
+type TextAskerProps = TextQuestion & {
+  initResponse: string;
+  onChange: (id: string, answer: string) => void;
+};
+
+type FormProps = HTMLAttributes<HTMLDivElement> & {
+  questions: Question[];
+  initResponses: { [key: string]: string };
+  onSubmit: (responses: Object) => void; // Takes in answer json and send it to db
+  id?: string; // id for the form HTML element to allow linking of external buttons
+};
+
+/**
+ * @summary Short text and Long Text Asker.
+ * Right now, there is no difference except placeholder. Do we want long answer to have multiline input functionality?
+ */
+const TextAsker = ({
+  id,
+  title,
+  type,
+  initResponse,
+  onChange,
+}: TextAskerProps) => {
+  const [answer, setAnswer] = useState(initResponse); // Replace with responses
+
+  return (
+    <TitledInput
```

Comment:
Might be better to use Text + Input instead of TitledInput because the title will be common across all questions